docs/pages/web/numberfield.js

```diff
@@ -556,6 +556,37 @@ function Example(props) {
           />
         </MainSection.Subsection>
 
+        <MainSection.Subsection
+          title="EnterKeyHint"
+          description={`The \`enterKeyHint\` prop presents to the user a more accurate action label for the enter key on virtual keyboards. These are the values for each use case:
+
+- "enter": typically used for inserting a new line
+- "done": typically used when there is nothing more to input and the input editor will be closed
+- "go":	typically used when taking the user to the target of the text they typed
+- "next":	typically used when taking the user to the next field that will accept text
+- "previous":	typically used when taking the user to the previous field that will accept text
+- "search":	typically used when taking the user to the results of searching for the text they have typed
+- "send":	typically used when delivering the text to its target
```

Comment:
"delivering the text to its target" is a bit awkward and unclear…perhaps something like "when submitting the input, such as an email or chat"?

Reply:
DONE